src/org/opendatakit/common/web/constants/BasicConsts.java

```diff
@@ -54,6 +54,7 @@
   public static final String RIGHT_BRACKET = "]";
   public static final String LEFT_PARENTHESIS = "(";
   public static final String RIGHT_PARENTHESIS = ")";
+  public static final String INVALID_CHARS = "[^a-zA-Z0-9\\s]";
```

Comment:
Replace all non alphanumeric character or non space character.

Reply:
We might want to make this more robust, because I think this will allow Unicode, which will break on some filesystems. I'm thinking we should only allow a-z, A-Z, 0-9, and hyphen, underscore, and space. And we should probably do the same thing when we write out the CSV. @dcbriccetti what do you think?

Reply:
This will only allow a-z, A-Z, 0-9 and space character in the title. You want to include hyphen and underscore in the list?

Reply:
Regex fail by @yanokwa 😭

Yeah, let's add hyphen and underscore!